src/Mgate/PersonneBundle/Form/Type/SexeType.php

```diff
@@ -21,8 +21,8 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choices' => array(
-                'M.' => 'Monsieur',
-                'Mme' => 'Madame',
+                'M.' => 'M.',
```

Comment:
It matches what Siaje does, so ok for me.

But it will empty all the existing data if they are edited again. So this should be associated with a doctrine migration. But isn't it some potential issues for a very little business value ?